src/__tests__/lib/exampleData.js

```diff
@@ -150,18 +150,17 @@ const randUserId: () => UserId = (mk => () => makeUserId(mk()))(
   makeUniqueRandInt('user IDs', 10000),
 );
 const userOrBotProperties = (args: UserOrBotPropertiesArgs) => {
-  const name = randString();
-  const capsName = name.substring(0, 1).toUpperCase() + name.substring(1);
   const user_id = args.user_id != null ? makeUserId(args.user_id) : randUserId();
+  const randName = randString();
   return deepFreeze({
     avatar_url: args.avatar_url ?? makeAvatarUrl(user_id.toString()),
 
     date_joined: `2014-04-${randInt(30)
       .toString()
       .padStart(2, '0')}`,
 
-    email: args.email ?? `${name}@example.org`,
-    full_name: args.full_name ?? `${capsName} User`,
+    email: args.email ?? `${randName}@example.org`,
+    full_name: args.full_name ?? `${randName} User`,
```

Comment:
```
example data [nfc]: Simplify random naming in makeUser
```

Tiny nit: I think this isn't quite NFC, in that when `args.full_name` isn't passed, the `full_name` property will no longer be guaranteed to have an uppercase first letter. (I think this is a good change, I just think it's slightly non-NFC in that way.)

Reply:
True. I'll tweak that commit message.